CHANGELOG.md

```diff
@@ -244,3 +244,17 @@ Fixes a bug in icon size for inline `<ChannelValue>` components.
 ## v0.0.1 (May 24, 2019)
 
 Initial beta release
+
+## v5.3.2 (September 30, 2021)
+
+### Updated
+
+-   Updated the import statements of material-ui components and icons to be default imports instead of named imports, so that the only components that are being used should be part of final bundle
+
+    ```typescript
+    /* Old import syntax */
+    import { ComponentName } from '@material-ui/core';
+
+    /* New import syntax */
+    import ComponentName from '@material-ui/core/ComponentName';
+    ```
```

Comment:
We can keep this entry pretty simple:
```suggestion
## v5.3.2 (unpublished)

### Updated

-   Import material-ui components via default imports instead of named imports to reduce overall bundle size.
```

And this should be at the top of the file instead of the bottom (most recent version appears first). You'll get the hang of the changelog updates once you do a few. We've been trying to follow the changelog format defined by Keep A Changelog: https://keepachangelog.com/en/1.0.0/

Reply:
@joebochill Updated the PR.